src/neoxp/IExpressNode.cs

```diff
@@ -40,6 +42,7 @@ enum CheckpointMode { Online, Offline }
         Task<IReadOnlyList<ExpressStorage>> ListStoragesAsync(UInt160 scriptHash);
         Task<IReadOnlyList<TokenContract>> ListTokenContractsAsync();
 
+        Task<int> PersistContractAsync(ContractState state, (string key, string value)[] storagePairs, ContractCommand.ContractForce? force);
```

Comment:
I'm not a fan of nullable enums. Can we add "None" as the default force parameter?

Reply:
I thought about that initially then assumed `nullable` things were the more standard C# approach. Will fix